runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -327,6 +327,51 @@ void J9::RecognizedCallTransformer::process_java_lang_StringUTF16_toBytes(TR::Tr
    treetop->insertAfter(TR::TreeTop::create(comp(), TR::Node::create(node, TR::treetop, 1, newCallNode)));
    }
 
+void J9::RecognizedCallTransformer::process_jdk_internal_util_ArraysSupport_vectorizedMismatch(TR::TreeTop* treetop, TR::Node* node)
+   {
+   TR::Node* a = node->getChild(0);
+   TR::Node* aOffset = node->getChild(1);
+   TR::Node* b = node->getChild(2);
+   TR::Node* bOffset = node->getChild(3);
+   TR::Node* length = node->getChild(4);
+   TR::Node* log2ArrayIndexScale = node->getChild(5);
+
+   TR::Node* lengthInBytes = TR::Node::create(node, TR::ishl, 2, length, log2ArrayIndexScale);
+   TR::Node* mask = TR::Node::create(node, TR::ior, 2,
+      TR::Node::create(node, TR::ishl, 2,
+         log2ArrayIndexScale,
+         TR::Node::iconst(node, 1)),
+      TR::Node::iconst(node, 3));
+   TR::Node* n = TR::Node::create(node, TR::iand, 2,
+      lengthInBytes,
+      TR::Node::create(node, TR::ixor, 2, mask, TR::Node::iconst(node, -1)));
+   // TODO: replace the aladd's in the following with generateDataAddrLoadTrees when off-heap memory changes come in
```

Comment:
@Spencer-Comin / @VermaSh  We should open up issue in OpenJ9 for this TODO

Reply:
Thanks @Spencer-Comin for adding the TODO. Created the issue https://github.com/eclipse-openj9/openj9/issues/16717